pkg/v1/remote/options.go

```diff
@@ -131,3 +134,23 @@ func WithContext(ctx context.Context) Option {
 		return nil
 	}
 }
+
+// WithParallel is a functional option for setting the parallelism of remote
+// operations performed by a given function. Note that not all remote
+// operations support parallelism.
+//
+// The default parallel is 4.
+func WithParallel(parallel int) Option {
```

Comment:
I called this WithJobs for gcrane, not sure if we want to keep this consistent: https://github.com/google/go-containerregistry/blob/ab3252bfe3ddb5e612349ff3fde459864cc6e8fb/cmd/gcrane/cmd/copy.go#L43

Reply:
WithJobs it is!